preferencesfx/src/main/java/com/dlsc/preferencesfx/model/PreferencesFxModel.java

```diff
@@ -182,7 +187,7 @@ public void saveSettingValues() {
   }
 
   /**
-   * Load all of the values of the settings using a {@link StorageHandler} and attaches a
+   * Load all of the values of the settings using a {@link StorageHandlerImpl} and attaches a
```

Comment:
should still be "StorageHandler", since if an own implementation of the `StorageHandler` is being used, it may not be called `StorageHandlerImpl`